packages/common/src/search/types/IHubSearchResult.ts

```diff
@@ -0,0 +1,110 @@
+import { IHubGeography } from "../..";
+
+/**
+ * Standardized light-weight search result structure, applicable to all
+ * types of search results - users, groups, content, events etc
+ */
+export interface IHubSearchResult {
+  /**
+   * Access level of the backing entity
+   */
+  access: "public" | "shared" | "org" | "private";
+  /**
+   * Identifier - could be a guid or a username
+   */
+  id: string;
+  /**
+   * Hub Family: "dataset" | "map" | "document" | "content"
+   * TODO: Can we create a type for this?
```

Comment:
Do you have pointers to the list of valid values? I tried to follow the code but was not quite clear

Reply:
I added a type for this, defined as `"app" | "content" | "dataset" | "document" | "event" | "feedback" | "initiative" | "map" | "other" | "site" | "solution" | "template" `
I _think_ that's the whole list based on `get-family.ts` and the `collections.ts` logic

Reply:
bah - turns out we actually _had_ an HubFamily type already... oddly typescript was ok with me exporting something with the same name... perhaps because it was exactly the same? Anyhow - now using the existing one